### PR TITLE
sci-biology/nilearn: new version

### PR DESCRIPTION
**Pull request by TheChymera**
newer versioned ref.

### CODE REVIEW

File: sci-biology/nilearn/nilearn-0.2.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+PYTHON_COMPAT=( python2_7 python3_4 )
+
+inherit distutils-r1 git-r3
+
+DESCRIPTION="Fast and easy statistical learning on NeuroImaging data"
+HOMEPAGE="http://nilearn.github.io/"
+SRC_URI=""
+EGIT_REPO_URI="git://github.com/nilearn/nilearn"
+EGIT_COMMIT="${PV}"
```
Comment (jlec) [inline sci-biology/nilearn/nilearn-0.2.4.ebuild]: Now versioned live ebuilds. Please use snapshots.

Comment (TheChymera) [inline sci-biology/nilearn/nilearn-0.2.4.ebuild]: How is it live? It's identified via a specific commit, making it as dead as any other source - no? In any case, where would the snapshots be? do you mean someting like https://github.com/stnava/ANTs/archive/v2.1.0.tar.gz ?

Comment (jlec) [inline sci-biology/nilearn/nilearn-0.2.4.ebuild]: > How is it live?

Pull your network cable and try to install it.

Comment (TheChymera) [inline sci-biology/nilearn/nilearn-0.2.4.ebuild]: It installed just fine. Problem solved?

Comment (jlec) [inline sci-biology/nilearn/nilearn-0.2.4.ebuild]: Either drop keywords or use a proper snapshot.

Comment (TheChymera) [inline sci-biology/nilearn/nilearn-0.2.4.ebuild]: what is a "proper" snapshot? Like the one I liked above?

Comment (TheChymera) [inline sci-biology/nilearn/nilearn-0.2.4.ebuild]: @jlec I really don't know what would make a better source. Prefferably I would like to use something from where most of the development happens (github) and not PIPy.

Comment (jlec) [inline sci-biology/nilearn/nilearn-0.2.4.ebuild]: Just download the tarball for that commit.

**Comment by TheChymera** [comment]
@jlec looking good?

**Comment by jlec** [comment]
> @jlec looking good?

Did you test the ebuild? Please do so.

**Comment by TheChymera** [comment]
@jlec tested it now

**Comment by TheChymera** [comment]
@jlec I tested this - and, as I said, it works. The log (afaict) does not indicate issues with nilearn. I cannot tell what is causing travis to fail. Is this ok? can you pull?

**Pull request merged**